### ISSUE TITLE
Smolen seems not to be implemented after 2N opening

### ISSUE THREAD

**Issue by ThorvaldAagaard** [EdwardPiwowar/BBA#1051]

**Comment by EdwardPiwowar** [comment]
Smolen is implemented. 3N shows both majors.
![image](https://github.com/user-attachments/assets/bf627f7d-2315-45c5-b244-080257f1e7c2)

**Comment by ThorvaldAagaard** [comment]
The headline was wrong and I must have forgotten to save it after update. When I started the report I did not notice than in the cc, puppet stayman was selected.

3N should NOT show both majors, unless there is implemented a way to transfer the suit to the strong hand.
There is no benefit otherwise, and just wrongsiding the contract.

**Comment by ThorvaldAagaard** [comment]
Yes, but the default is 3N is natural (Want to play 3N even if partner has a Smolen), so we need the option for those who does not follow standard. And that is my point, that when you make a change like this, you must consider how to rightside the hand.
